source/aws-bootstrap-kit/test/account.test.ts

```diff
@@ -15,14 +15,15 @@ limitations under the License.
 */
 
 import { expect as expectCDK, haveResource, countResourcesLike } from "@aws-cdk/assert";
```

Comment:
FYI, the assert seems to be deprecated in favor of assertions

Reply:
Yes, we need to migrate to the new assertions. Will create an issue, thank you.